api/src/main/java/org/apache/cloudstack/api/ApiConstants.java

```diff
@@ -722,6 +722,7 @@
     public static final String VIRTUAL_SIZE = "virtualsize";
     public static final String NETSCALER_CONTROLCENTER_ID = "netscalercontrolcenterid";
     public static final String NETSCALER_SERVICEPACKAGE_ID = "netscalerservicepackageid";
+    public static final String INCLUDE_ROUTER_HEALTH_CHECK_RESULTS = "includehealthcheckresults";
```

Comment:
Can it be renamed to 'healthcheckresults' instead?

Reply:
:+1: